README.md

```diff
@@ -18,14 +18,14 @@ With Maven you can include the recent library version as dependency with:
 <dependency>
   <groupId>life.qbic</groupId>
   <artifactId>data-model-lib</artifactId>
-  <version>2.8.2</version>
+  <version>2.9.0</version>
```

Comment:
In theory this should be updated as well but I'll prepare a PR that includes the version directly from the Pom.xml soon :)